cinch/library/jenkins_plugin.py

```diff
@@ -0,0 +1,826 @@
+#!/usr/bin/python
```

Comment:
I wanted to keep the file as close to upstream as possible, so I didn't change anything other than the necessary lines for bug fixing. Changing the shebang wouldn't be a problem, since 

Do you mean a Python submodule or a git submodule?